src/main/java/org/broadinstitute/consent/http/cloudstore/GCSService.java

```diff
@@ -63,32 +58,6 @@ public void setConfig(StoreConfiguration config) {
         this.config = config;
     }
 
-    //method only called by whitelist service 
-    //deprecated, to be removed along with whitelist classes
-    public GenericUrl postWhitelist(String content, String fileName) {
-        BlobId blobId = BlobId.of(config.getBucket(), "whitelist/" + fileName);
-        BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("text/plain").build();
-        Blob blob = storage.create(blobInfo, content.getBytes());
-        return new GenericUrl(blob.getMediaLink());
```

Comment:
There are some more unused imports from changes to this file. Lines 3-4 can be removed.